.changeset/dirty-teams-ask.md

```
@@ -0,0 +1,5 @@
+---
+'@mastra/client-js': patch
+---
+
+Added thread-specific task hydration to agent controller session state requests.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)